.travis.yml

```diff
@@ -1,19 +1,29 @@
 language: julia
 os:
-    - linux
+  - linux
 julia:
-    - 1.0
-    - 1.2
-    - nightly
+  - 1.0
+  - 1.2
+  - 1.3
+  - nightly
 matrix:
   allow_failures:
     - julia: nightly
   fast_finish: true
 notifications:
   email: false
+
+jobs:
+  include:
+    - stage: "Documentation"
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate(); Pkg.build()'
```

Comment:
I guess it's this line saying that you get the dependencies related to this particular package. If you need something else, e.g. Documenter.jl which was in `REQUIRE` earlier, that needs to specifically require using `docs/Project.toml`. So basically we don't want that our package depends on Documenter.jl, but the documentation can depend on that.

Reply:
Or in other words, if you run `julia --activate=docs/` and try to run the example in there, it should fail because the packages are not installed into that virtual environment.

Reply:
Yes, that's why I felt like an idiot because I couldn't figure out why it's failing.